aten/src/ATen/test/CMakeLists.txt

```diff
@@ -21,6 +21,10 @@ list(APPEND ATen_CPU_TEST_SRCS
   ${CMAKE_CURRENT_SOURCE_DIR}/verify_api_visibility.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/tbb_init_test.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/weakref_test.cpp)
+if (NOT USE_ROCM)
+  list(APPEND ATen_CPU_TEST_SRCS
+  ${CMAKE_CURRENT_SOURCE_DIR}/extension_backend_test.cpp)
```

Comment:
Why exclude it when USE_ROCM?

Reply:
Something (probably something to do with hipify) causes this to fail on rocm. I haven't found exactly why, but don't think this should block this diff. I'll add a comment.

Reply:
But this file shouldn't be hipified since it's just cpu file (and also is not going to be compiled by the hcc compiler). Could you paste the error you have seen?

Reply:
Issue identified, it's indeed unrelated to rocm (see https://github.com/pytorch/pytorch/pull/16606). The reason this issue has been only triggered in rocm CI is likely because they the only ones using gcc 7/clang 7 among all the pytorch CI.
Please absorb https://github.com/pytorch/pytorch/pull/16606/commits/10cd89294f10bf6343054c7c508b0154c3f83a62 into this PR to add back the test into rocm builds.

Reply:
thanks, much appreciated